doc/getting_started.rst

```diff
@@ -8,10 +8,11 @@ Creating a basic Gallery
 ========================
 
 This section describes how to set up a basic gallery for your examples
-using ``sphinx-gallery``. You'll do the following.
+using the Sphinx extension Sphinx-Gallery, which will do the following:
 
-* Automatically generate sphinx rST for your examples, along with Jupyter
-  notebooks of each that users can download.
+* Automatically generate Sphinx rST for your example ``.py`` files, along with
+  ``.ipynb`` (Jupyter notebook) and ``.py`` files of each example that users 
+  can download.
```

Comment:
* Automatically generate Sphinx rST out of your ``.py`` example files. 
  The rendering of the resulting rST will provide the users with 
  ``.ipynb`` (Jupyter notebook) and ``.py`` files of each example to be downloaded.